### ISSUE TITLE
Holo Cube and Battery puzzle

### ISSUE THREAD

**Issue by voelkerb** [ubilab-ws21/esclab#10]
Seems to be not ready yet?

I am not 100% sure about the flow of your puzzle. 
Is it: -> solve HoloCube -> get code for safe -> take out battery -> load battery -> put back?

- [ ] HoloCube not working yet
- [ ] Code needs to open safe (test this)
- [x] Battery loading HW
- [x] Full battery detection
- [ ] Room must indicate that power is now back
  - [ ] Text to speech 
  - [ ] Lights on

I see that the team uses GitHub issues a lot, so we can maybe discuss things not working here.

**Comment by archerindigo** [comment]
The flow is correct, but the safe should automatically be unlocked after the 2048 game, as Philips found it more interesting.

As for control room's power back on, isn't it a task for environment team?

**Comment by voelkerb** [comment]
okay, got it. Yeah that was actually the plan, but as there is no environment team anymore, we need to adapt ;)

**Comment by archerindigo** [comment]
So, is there some sort of commands to remote control the lights?

**Comment by archerindigo** [comment]
If tts is to be used during power restoration as well, I suggest this sentence: "Emergency power activated"

**Comment by voelkerb** [comment]
Yeah the commands to control the lighting should be the same as last year and is (hopefully) documented by the environment group of last term. I will try to repair the hardware until next week. We need to communicate with the operator @sboettcher if you should send the commands for the lighting and tts or if this is done in the games control flow

**Comment by sboettcher** [comment]
Anything that happens between puzzles or between puzzle sequences we can do from the operator, otherwise it needs to be handled by the puzzle. In this case it sounds like the lights should come on and a tts message should be played after the puzzle is "solved", so we can do it from the operator logic.

**Comment by voelkerb** [comment]
@sboettcher is the power back on tts already implemented - I can't remember it was in our discussion this evening? Same for light on if battery is charged and placed in box.

**Comment by sboettcher** [comment]
Ah no good point. Can I just add the TTS+Lights between this and radio puzzle then?

**Comment by voelkerb** [comment]
We can also do:
1 solarstorm -> everything is black
2 reboot sound + red lights in the room 
3 when door is open -> TTS that backup battery needs reloading
4 when battery puzzle is solved, room back to white color + TTS "Back on emergency power" -> radio goes on
